code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm

```diff
@@ -567,6 +568,82 @@
 	R.verbs |= /mob/living/silicon/robot/proc/ex_reserve_refill
 	..()
 
+// SKYVAAL ADDED SERVICE BORG CODE
+/obj/item/weapon/robot_module/robot/servborg
+	name = "Service Hound Module"
+	can_buckle = 1
+	sprites = list(
+					"Chef-borg" = "servborg",
+					)
+	channels = list("Service" = 1)
+	can_be_pushed = 0
+
+/obj/item/weapon/robot_module/robot/servborg/New(var/mob/living/silicon/robot/R)
+
+	// Dogborg Standard
+	src.modules += new /obj/item/weapon/dogborg/jaws/small(src)
+	src.modules += new /obj/item/device/dogborg/boop_module(src)
+
+	//Service Module Copy/Paste
+	src.modules += new /obj/item/weapon/gripper/service(src)
+	src.modules += new /obj/item/weapon/reagent_containers/glass/bucket(src)
+	src.modules += new /obj/item/weapon/material/minihoe(src)
+	src.modules += new /obj/item/weapon/material/knife/machete/hatchet(src)
+	src.modules += new /obj/item/device/analyzer/plant_analyzer(src)
+	src.modules += new /obj/item/weapon/storage/bag/plants(src)
+	src.modules += new /obj/item/weapon/robot_harvester(src)
+	src.modules += new /obj/item/weapon/material/knife(src)
+	src.modules += new /obj/item/weapon/material/kitchen/rollingpin(src)
+	src.modules += new /obj/item/device/multitool(src) //to freeze trays
+
+	var/obj/item/weapon/rsf/M = new /obj/item/weapon/rsf(src)
+	M.stored_matter = 30
+	src.modules += M
+
+	src.modules += new /obj/item/weapon/reagent_containers/dropper/industrial(src)
+
+	var/obj/item/weapon/flame/lighter/zippo/L = new /obj/item/weapon/flame/lighter/zippo(src)
+	L.lit = 1
+	src.modules += L
+
+	src.modules += new /obj/item/weapon/tray/robotray(src)
+	src.modules += new /obj/item/weapon/reagent_containers/borghypo/service(src)
+	src.emag = new /obj/item/weapon/reagent_containers/food/drinks/bottle/small/beer(src)
+
+	var/datum/reagents/F = new/datum/reagents(50)
+	src.emag.reagents = F
+	F.my_atom = src.emag
+	F.add_reagent("beer2", 50)
+	src.emag.name = "Mickey Finn's Special Brew"
+
+	var/datum/matter_synth/water = new /datum/matter_synth()
+	water.name = "Water reserves"
+	water.recharge_rate = 0 // Recharging water for plants - hehe drooly borg
+	water.max_energy = 1000
+	water.energy = 0
+	R.water_res = water
+	synths += water
+
+	var/obj/item/device/dogborg/tongue/T = new /obj/item/device/dogborg/tongue(src)
+	T.water = water
+	src.modules += T
+
+	// PLACEHOLDER UNTIL MICROWAVE BELLY SORTED
+	var/obj/item/device/dogborg/sleeper/K9/B = new /obj/item/device/dogborg/sleeper/K9(src) //Eat criminals. Bring them to the brig.
```

Comment:
I don't think this is needed for what's s'posed to be a borg to work in the kitchen.

Reply:
Might want to comment it out until you actually get the thing working.

Reply:
Well, its mostly a placeholder cuz you know how people love their dogborgs.
I'm sitting down and figuring it out for now, and out of the bellies we already have, its the one that makes the most sense...
I _could_ comment it out, or see about hiding it behind the EMAG for now?
Chloral Hydrate + Brigbelly = Kidnappy fun :P

Reply:
So this whole thing's still an WIP? Otherwise, if this gets merged, people are going to probably be confused about the name.

Reply:
Hmm, rename the module then, too? 
and I meant the brig-belly is a placeholder :P